### ISSUE TITLE
IDK WHY YOU DONT HAVE A GOOD DOCS

### ISSUE THREAD

**Issue by amd-86** [tiangolo/fastapi#3358]
on an invalid path i am getting ``{"detail":"Not Found"}`` 404 error how can i customized this errrorr i found no way please

**Comment by chbndrhnns** [comment]
I found this in the docs: https://fastapi.tiangolo.com/tutorial/handling-errors/#override-the-default-exception-handlers

**Comment by Kludex** [comment]
@amd-86 That tone is not appreciated in this community. That's not how you ask for help.

**Comment by amd-86** [comment]
> @amd-86 That tone is not appreciated in this community. That's not how you ask for help.

you are right Kludex i apologize for that, i was mad and idk how i said it pardon, cheers.
